backend/src/v2/compiler/argocompiler/container.go

```diff
@@ -30,9 +30,9 @@ import (
 const (
 	volumeNameKFPLauncher    = "kfp-launcher"
 	volumeNameCABundle       = "ca-bundle"
-	DefaultLauncherImage     = "ghcr.io/kubeflow/kfp-launcher:2.4.0"
+	DefaultLauncherImage     = "ghcr.io/kubeflow/kfp-launcher:latest"
```

Comment:
Can you please add a comment above mentioning that the latest tag will be replaced by with a pinned version at release time?

Reply:
@chensun the goal is to pin the version from the manifests, seen [here](https://github.com/kubeflow/pipelines/pull/11533/files#diff-b8ec42e65046bed5636c791664edec2351cef100ac7d2dac2ae1d20c8acf1852R112-R115) 

Apologies for the large diff on the yaml, the only change is what is highlighted above, the rest is just fixing indenting. The idea is to keep this `latest` and update the version in the manifests so we can build all these images (including api server, etc.) all at once during releases. Let me know what you think about this.

Reply:
If there is concern about having `latest` in code, in a release version, we can also make the driver/launcher env vars a hard requirement, and throw an error when it is not provided, and simply not have a default hardcoded tag at all

Reply:
The concern is just that when a developer/maintainer saw this in code, without the context or digging into the change history, they might be alerted by the "latest" usage. A comment would help clarify that latest is not what users get with the released version.

Reply:
done!